src/MapWindow/GlueMapWindowItems.cpp

```diff
@@ -131,10 +135,12 @@ GlueMapWindow::ShowMapItems(const GeoPoint &location,
     return false;
   }
 
-  ShowMapItemListDialog(list,
+  doRefreshMapList = ShowMapItemListDialog(list,
                         UIGlobals::GetDialogLook(), look, traffic_look,
                         final_glide_bar_renderer.GetLook(), settings,
                         glide_computer != nullptr
                         ? &glide_computer->GetAirspaceWarnings() : nullptr);
+  }while(doRefreshMapList);
```

Comment:
Bad coding style

Reply:
Please, I want to learn. What is better

Reply:
Two spaces missing.  `} while (doRefreshMapList);`
Read the developer manual's section "Code style".